tools/onnx_subgraph/README.md

```diff
@@ -0,0 +1,62 @@
+# onnx_autosubgraph
+onnx-subgraph tool provides model auto partitionioning of onnx model to several sub models by 
+operator, performance and model size limitations, with the order and input / output names of 
+sub models.
+
+# How to build the onnx-subgraph
+## OS environment dependence
+     1. ubuntu >=20.04
+     2. GCC >= 9.4.0
+     3. cmake >= 3.10
+     4. python >= 3.8
+     5. apt-get install libprotobuf-dev protobuf-compiler libjsoncpp-dev
+
+## Python packages dependence
+    onnx                         1.16.0
+    onnxruntime                  1.18.1
+    onnxsim                      0.4.36
+    torch                        2.3.1
+    scikit-image
+    scikit-learn
+    pandas
+    tqdm
+    
+## building the onnx-subgraph
+    '''
+    cd onnx-subgraph
+    mkdir build & cd build
+    cmake .. & make
+    '''
+    we can get following output at './build'
+        '''
+        scripts
+        ├── extract_onnx.py
+        └── test_model_download.sh
+        └── subgraphs_ios.txt
+        '''
```

Comment:
rendering isn't what I expected. sorry for this.
![image](https://github.com/user-attachments/assets/1b27c36b-5f20-478b-9240-178ca9a25d06)
plz try ` (keyboard with ~)not '

Reply:
plz run `view file` and see rendered output and fix as you wish.
I can't suggest which is too compliate to explain.